src/components/LatestRatings.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { gql } from 'apollo-boost';
+import { Query } from 'react-apollo';
+
+const Hero = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  min-height: calc(100vh - 7.375rem);
+  text-align: center;
+`;
+
+const GET_LATEST_RATINGS_QUERY = gql`
+  query GET_LATEST_RATINGS_QUERY($max: Int) {
+    getLatestRatings(max: $max) {
+      _id
+      author
+      caller
+      callType
+      message
+      score
+      createdAt
+      number {
+        number
+      }
+    }
+  }
+`;
+
+function Comment({ rating }) {
```

Comment:
This `Comment` component could have its own file as we will use them on other pages too. It's not a comment though but a review or rating, so `Rating` would probably be a better name.